integration-cli/docker_cli_prune_unix_test.go

```diff
@@ -89,3 +90,51 @@ func (s *DockerDaemonSuite) TestPruneImageDangling(c *check.C) {
 	c.Assert(err, checker.IsNil)
 	c.Assert(strings.TrimSpace(out), checker.Not(checker.Contains), id)
 }
+
+func (s *DockerSuite) TestPruneContainerUntil(c *check.C) {
```

Comment:
Can we have a test for `--until=5s`? (IMO we should allow some suffix like "ago". i. e. support `--until="5s ago"` as well)

See also: https://github.com/docker/docker/blob/master/api/types/time/timestamp_test.go